src/TouchCarousel.js

```diff
@@ -55,22 +55,37 @@ class TouchCarousel extends React.PureComponent {
     this.grabbing = false
     this.tracingTouchId = null
     this.isMovingCross = null
+    this.isMouseDown = false
+    this.mouseTouchId = 0
   }
 
   componentDidMount () {
+    document.addEventListener('mousemove', this.onMouseMove)
+    document.addEventListener('mouseup', this.onMouseUp)
     this.autoplayIfEnabled()
   }
 
   componentWillUnmount () {
+    document.removeEventListener('mousemove', this.onMouseMove)
+    document.removeEventListener('mouseup', this.onMouseUp)
```

Comment:
Maybe name like `onDocumentMouseup`? To distinguish from other handlers on the container.

Reply:
Good point, will fix.